docs/astro/sql/operators/raw_sql.rst

```diff
@@ -0,0 +1,27 @@
+====================
+run_raw_sql operator
+====================
+
+It allows users to declare any SQL statement using the Astro SDK templating available in transform. By default this operator returns ``None`` but it also empowers users to define the task output (e.g. a list with the row results, etc) by using the handler argument.
+
+
+Create Snowflake table using ``run_raw_sql`` without ``handler`` argument i.e this will return ``None``
+
+.. literalinclude:: ../../../../example_dags/example_snowflake_partial_table_with_append.py
+    :language: python
+    :start-after: [START howto_run_raw_sql_snowflake_1]
+    :end-before: [END howto_run_raw_sql_snowflake_1]
+
+Use above task in your DAG
+
+.. literalinclude:: ../../../../example_dags/example_snowflake_partial_table_with_append.py
```

Comment:
@pankajastro do we need this example separately? or maybe we can combine it with the previous example? WDYT?

Reply:
hmm, consolidated in one